docs/index.md

```diff
@@ -257,6 +257,48 @@ connection "aws_account_z" {
 
 Using named profiles allows Steampipe to work with your existing CLI configurations, including SSO and using role assumption.
 
+### AWS Profile Credentials with IAM Roles
+You may have an IAM User in an AWS account used to hop-off to other accounts via assume role. In this case the AWS Credential files looks like this:
```

Comment:
```suggestion
You may have credentials in a profile used to assume a role in the same or different AWS account.
```

My suggestions are based off of the following assumptions (but I haven't verified), so if any of these are incorrect please let me know:
- Do the creds in the starting profile need to be from an user, i.e., using an access key pair? Or could they be any type of supported profile credentials (assuming this case)?
- Does the role need to be in another account, or could it be to a role inside the same account (assuming this case)?